ui/packages/consul-ui/app/components/certificate/index.scss

```diff
@@ -0,0 +1,26 @@
+.certificate {
+  display: flex;
+  button.visibility {
+    height: fit-content;
+    padding-top: 4px;
+    margin-right: 4px;
+    cursor: pointer;
+  }
+  button.hide::before {
+    @extend %with-visibility-hide-icon, %as-pseudo;
+  }
+  button.show::before {
+    @extend %with-visibility-show-icon, %as-pseudo;
+  }
+  div.key {
+    background-color: var(--gray-050);
+    overflow-wrap: anywhere;
+  }
+  hr {
+    border: 3px dashed var(--gray-300);
+    background-color: $white;
+    width: 10vw;
```

Comment:
No biggie, wondered if this should be a fixed width so it doesn't change when I resize my window?

Reply:
💯